index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Marketing Agency</title>
+  </head>
+
+  <!-- webpage header section with navbar -->
+  <body>
+    <header class="header">
+      <h1>Hori<span class="seo">seo</span>n</h1>
+      <nav>
+        <ul>
+          <li>
+            <a href="#search-engine-optimization">Search Engine Optimization</a>
+          </li>
+          <li>
+            <a href="#online-reputation-management"
+              >Online Reputation Management</a
+            >
+          </li>
+          <li>
+            <a href="#social-media-marketing">Social Media Marketing</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <!-- website banner image -->
+    <figure class="hero"></figure>
+
+    <!-- main starts here containing three sections -->
+    <main class="content">
+      <section id="search-engine-optimization" class="main-section">
+        <img
+          src="./assets/images/search-engine-optimization.jpg"
+          class="float-left"
+          class="float-left"
```

Comment:
The class has been added in twice here.

Reply:
Hi Chelsea, I didn't notice. I will amend that. Thank you.